e2e/app/page/admin-user-profile-info.ts

```diff
@@ -133,24 +133,25 @@ export default class UserProfileInfo extends AuthenticatedPage {
     return Textbox.findByName(this.page, { dataTestId: DataTestIdAlias.ContactEmail });
   }
 
-  getFreeCreditsUsedInput(): Textbox {
+  getInitialCreditsUsedInput(): Textbox {
     return Textbox.findByName(this.page, { dataTestId: DataTestIdAlias.InitialCreditsUsed });
   }
 
-  // get the free credits used value
-  async getFreeCreditMaxValue(): Promise<string> {
-    const freeCreditsMax = this.getFreeCreditsUsedInput();
-    const freeCreditsMaxValue = await freeCreditsMax.getProperty<string>('value');
-    const regex = new RegExp(/(?<=of ).*(?= limit)/);
-    return regex.exec(freeCreditsMaxValue)[0];
+  // get the limit value from the initial credits used field
+  async getInitialCreditUsedValue(): Promise<string> {
```

Comment:
This doesn't get the amount used - it gets the limit

Reply:
Ok! will update it. Thanks!

Reply:
done!